java/Ice/cancellation/README.md

```diff
@@ -0,0 +1,32 @@
+# Ice Cancellation
+
+The Cancellation demo shows how to cancel an invocation using the Java CancellationException. It also shows a related
```

Comment:
You meant:
shows how to cancel an invocation by interrupting the thread that waits for the response?